app/services/propertylinking/PropertyLinkingService.scala

```diff
@@ -72,7 +72,7 @@ class PropertyLinkingService @Inject()(
     for {
       propertyHistory <- propertyLinkConnector.getPropertyHistory(uarn)
       earliestStartDate: Option[LocalDate] = propertyHistory.history
-        .find(pv => pv.listType == ListType.CURRENT && pv.valuationStatus == ValuationStatus.CURRENT)
+        .find(_.propertyLinkEarliestStartDate.isDefined)
```

Comment:
is there a test that covers this scenario? might be worth adding one if it doesn't already exist

Reply:
I have added new testcase Please check